hardware/Makefile

```diff
@@ -286,7 +286,12 @@ hw_project: hw_project_start action_hw
 config: hw_project
 
 image: .hw_project_done
-	@if [ `echo "$(USE_PRFLOW)" | tr a-z A-Z` = "TRUE" ]; then \
+ifeq ($(USE_PRFLOW),TRUE)
+	@echo -e "[BUILD IMAGE.........] start `date +"%T %a %b %d %Y"`\n"
+	snap-cloud-build ${SNAP_ROOT}/snap_env.sh
+	@echo -e "[BUILD IMAGE.........] end `date +"%T %a %b %d %Y"`\n"
+else
+  @if [ `echo "$(USE_PRFLOW)" | tr a-z A-Z` = "TRUE" ]; then \
 		echo; echo -e "Error: Makefile target $* not allowed for PR flow!"; exit -1; \
 	fi
 	@echo -e "[BUILD IMAGE.........] start `date +"%T %a %b %d %Y"`\n"
```

Comment:
@kenhill These lines are not required anymore.
I did create pull request #498 into this branch containing a suggested change for this.
Please merge this change into this branch or create a different version if you don't like my change.